src/PrivateDns/PrivateDns/VirtualNetworkLinks/SetAzurePrivateDnsVirtualNetworkLink.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
For this cmdlet, it looks like you are running an "Update" rather than "CreateOrUpdate" SDK call. In this case, you might consider making this cmdlet an "Update" rather than a "Set".  For a "Set", you automatically set all properties to default that a user does not explicitly specify, whereas in an update, only the properties that are provided by the user are updated (the rest stay the same).  Additionally, rather than a SwitchParameter for values that are either true or false (which you always use in Sets and News), you would use a boolean parameter so a user can explicitly set a value, and nothing is changed if the parameter is not specified.

Reply:
changed it as u mentioned.